Library/RadarIO.Xamarin.Lib/RadarSDK.cs

```diff
@@ -77,11 +77,10 @@ public enum RadarRouteUnits
 
     public class RadarRoutes
     {
-#if MONOANDROID
+        /// <summary>
+        /// iOS only uses distance property
```

Comment:
This isn't iOS only.  Is there a missing Android caller?

Reply:
Why doesn't Android need this?

Reply:
I think my grammar in the comment has caused some confusion here. This property exists for both iOS and Android. Android has the full property, but for iOS only the distance field of this property is used, the other two are left null. I'm not sure why, it's just how the iOS sdk is